docs/INSTALL_GUIDE.md

```diff
@@ -0,0 +1,156 @@
+# Installation
+
+Telegraf compiles to a single static binary, which makes it easy to install.
+Both InfluxData and the community provide for a wide range of methods to install
+Telegraf from. For details on each release, view the [changelog][] for the
+latest updates and changes by version.
+
+[changelog]: /CHANGELOG.md
+
+There are wide range of places to obtain Telegraf from:
+
+* [Binary downloads](#binary-downloads)
+* [Homebrew](#homebrew)
+* [InfluxData Linux package repository](#influxdata-linux-package-repository)
+* [Official Docker images](#official-docker-images)
+* [Helm charts](#helm-charts)
+* [Nightly builds](#nightly-builds)
+* [Build from source](#build-from-source)
+* [Custom builder](#custom-builder)
+
+## Binary downloads
+
+Binary downloads for a wide range of architectures and operating systems are
+available from the [InfluxData downloads][] page or from the
+[GitHub Releases][] page.
+
+[InfluxData downloads]: https://www.influxdata.com/downloads
+[GitHub Releases]: https://github.com/influxdata/telegraf/releases
+
+## Homebrew
+
+A [Homebrew Formulae][] for Telegraf that updates after each release:
```

Comment:
```suggestion
A [Homebrew Formula][] for Telegraf that updates after each release:
```
`Formulae` is the plural form, though I'm not sure if using `Formula` it for a singular term like this is "correct" since it seems to be a technical term, your call

Reply:
Yeah I debated on this one, let's switch it to the singular.